tests/app/va/va_profile/test_va_profile_client_for_profile_v3.py

```diff
@@ -16,10 +18,59 @@
     VAProfileNonRetryableException,
     VAProfileRetryableException,
 )
-from app.va.va_profile.va_profile_client import CommunicationChannel
+from app.va.va_profile import VAProfileClient
+from app.va.va_profile.va_profile_client import CommunicationChannel, VALID_PHONE_TYPES_FOR_SMS_DELIVERY
+from app.va.va_profile.va_profile_types import Telephone
 
-from tests.app.factories.feature_flag import mock_feature_flag
 from tests.app.conftest import MOCK_VA_PROFILE_URL
+from tests.app.factories.feature_flag import mock_feature_flag
+
+
+def telephone_entry(
+    create_date=datetime.today(), phone_type='MOBILE', classification_code='0', name_for_debugging='test mobile phone'
+):
+    area_code = random.randint(100, 999)
+    phone_number = random.randint(100000, 9999999)
+    return {
+        'createDate': f'{create_date}',
+        'updateDate': f'{create_date}',
+        'txAuditId': '6706b496-d727-401f-8df7-d6fc9adef0e7',
+        'sourceSystem': name_for_debugging,
+        'sourceDate': '2022-06-09T15:11:58Z',
+        'originatingSourceSystem': 'VETSGOV',
+        'sourceSystemUser': '1012833438V267437',
+        'effectiveStartDate': '2022-06-09T15:11:58Z',
+        'vaProfileId': 1550370,
+        'telephoneId': 293410,
+        'internationalIndicator': False,
+        'phoneType': phone_type,
+        'countryCode': '1',
+        'areaCode': f'{area_code}',
+        'phoneNumber': f'{phone_number}',
+        'classification': {'classificationCode': classification_code},
+    }
+
+
+@pytest.fixture(scope='function')
+def mock_va_profile_client(mocker, notify_api):
+    with notify_api.app_context():
+        mock_logger = mocker.Mock()
+        mock_ssl_key_path = 'some_key.pem'
+        mock_ssl_cert_path = 'some_cert.pem'
+        mock_statsd_client = mocker.Mock()
+        mock_va_profile_token = mocker.Mock()
+
+        client = VAProfileClient()
+        client.init_app(
+            logger=mock_logger,
+            va_profile_url=MOCK_VA_PROFILE_URL,
+            ssl_cert_path=mock_ssl_cert_path,
+            ssl_key_path=mock_ssl_key_path,
+            va_profile_token=mock_va_profile_token,
+            statsd_client=mock_statsd_client,
+        )
+
+        return client
```

Comment:
This method is in `tests/app/conftest.py`, so it should be available to these tests.
```suggestion
```

Reply:
@AdamKing0126